src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/createbundle/CreateBundleAboutToStartHandler.java

```diff
@@ -4,8 +4,6 @@
 import static uk.gov.hmcts.reform.sscs.model.AppConstants.DWP_DOCUMENT_EVIDENCE_FILENAME_PREFIX;
```

Comment:
Do we still require this about to Submit callback?

Reply:
Yes. It sends an error if mandatory files are missing. It also updates the case. 
It's too late to update the case on the Submitted - you can do it - but you'd have to call an update event.

The call to `serviceRequestExecutor.post(callback, bundleUrl + CREATE_BUNDLE_ENDPOINT);` could also return an error or warning (maybe rare that it could occur) that we will miss on the ccd ui as it's on the submitted event. Perhaps I could add some logging if that call returns any errors or warnings